dbt/adapters/athena/connections.py

```diff
@@ -157,16 +158,13 @@ def open(cls, connection: Connection) -> Connection:
                 ),
             )
 
-            connection.state = "open"
+            connection.state = ConnectionState.OPEN
             connection.handle = handle
 
         except Exception as e:
-            logger.debug("Got an error when attempting to open a Athena "
-                         "connection: '{}'"
-                         .format(e))
+            logger.exception("Got an error when attempting to open a Athena connection")
```

Comment:
Could you also add the returned exception in logger.exception? As it is now it will just raise a unique generic error!

Reply:
According to this [documentation](https://docs.python.org/3/library/logging.html#logging.exception), the exception should be appended to the message.

Reply:
```python
>>> try:
...     1/0
... except Exception:
...     logging.exception("An error occured while operating")
... 
ERROR:root:An error occured while operating
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ZeroDivisionError: division by zero
```

Reply:
the stack error trace shown always  the error of course but when I log something I like to see the actual error, those generic messages are really ugly to track them down. So please could you add something like that?
f`Got an error when attempting to open a Athena connection due to {err}`

Reply:
Ok I can definitely add this. :)